OpenSim/Simulation/Model/Geometry.h

```diff
@@ -232,6 +228,7 @@ class OSIMSIMULATION_API LineGeometry : public Geometry
         upd_end_point() = aPoint2;
     }
 
+protected:
     /** Virtual method to map LineGeometry to SimTK::Array of
     SimTK::DecorativeGeometry.  Appearance, Transforms are handled by base
     Geometry class. */
```

Comment:
@aseth1 Making the Super call requires some implementation in the base class which can be empty but has the negative side effect that when you add a new kind of Geometry you don't know what to implement. I'm leaning toward keeping it as clear as possible with a pure virtual in the base class. I think this a place to apply the Impl naming convention, so we can leave the method pure virtual and change the name into implementCreateDecorativeGeometry() per #648